<launch>

<!--	<node pkg="joy_test" name="motion_node_final" type="motion_node_final" output="screen"/> -->
<node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
    <remap from="image" to="/usb_cam/image_raw"/> 
<!--		<remap from="image" to="/image_raw"/> -->
<!--		<remap from="image" to="/camera/image_raw"/>  -->
    <param name="autosize" value="true" />
  </node>

</launch>
